pkg/api/v1beta3/types.go

```diff
@@ -623,7 +629,7 @@ type HTTPGetAction struct {
 // TCPSocketAction describes an action based on opening a socket
 type TCPSocketAction struct {
 	// Required: Port to connect to.
-	Port util.IntOrString `json:"port" description:"number of name of the port to access on the container"`
+	Port util.IntOrString `json:"port" description:"number or name of the port to access on the container; number must be in the range 1 to 65536; name must be a IANA_SVC_NAME"`
```

Comment:
on all of these: ports are 1-65535, not 65536.  we are using inclusive ranges elsewhere.